tests/integ/test_source_dirs.py

```diff
@@ -0,0 +1,40 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+from __future__ import absolute_import
+
+import os
+
+from sagemaker.pytorch.estimator import PyTorch
+from tests.integ import DATA_DIR, PYTHON_VERSION
+
+
+def test_source_dirs(sagemaker_session, tmpdir):
+    source_dir = os.path.join(DATA_DIR, 'pytorch_source_dirs')
+    lib = os.path.join(str(tmpdir), 'alexa.py')
```

Comment:
any reason not to put it in a separate directory? 
i think it would be closer to a real life use case.